src/index.ts

```diff
@@ -187,6 +197,20 @@ function getLatestSliderDate(cb: (date: Date) => void) {
   });
 }
 
+// TODO: Finish
+function getLatestMeteosatDate() {
```

Comment:
Yeah, I was just trying to get a minimal runnable skeleton in place before implementing all the stuff, and since this is my first time writing TypeScript I wanted to make sure I grasped the basic concepts.